src/fsharp/FSharp.Core/fslib-extra-pervasives.fsi

```diff
@@ -92,15 +92,31 @@ module ExtraTopLevelOperators =
     /// <summary>Converts the argument to byte.</summary>
     /// <remarks>This is a direct conversion for all 
     /// primitive numeric types. For strings, the input is converted using <c>Byte.Parse()</c> on strings and otherwise requires a <c>ToByte</c> method on the input type.</remarks>
-    [<CompiledName("ToByte")>]
-    val inline uint8       : value:^T -> byte       when ^T : (static member op_Explicit : ^T -> byte)       and default ^T : int        
+    [<CompiledName("ToUInt8")>]
```

Comment:
I have reverted this change.  The compiled name should really always have been "ToUInt8", but you are correct, we won't change it now for backwards-compat reasons.